python-regression/util/test_logic/api_test_logic.py

```diff
@@ -47,12 +47,12 @@ def check_neighbors(step, node):
     for i in response:
         expected_neighbors = step.hashes
         if type(response[i]) != int:
-            for x in range(len(response[i])):    
+            for x in range(len(response[i])):
                 if expected_neighbors[0]['neighbors'] == response[i][x]['address']:
                     contains_neighbor[0] = True
                 if expected_neighbors[1]['neighbors'] == response[i][x]['address']:
                     contains_neighbor[1] = True
```

Comment:
This function is only for 2 neighbors?

Reply:
Currently yes, I'm going to do a separate PR to replace the neighbor logic, but it seemed inappropriate to tag it on to this PR.